### PR TITLE
Translation languages are supported in the README file and all matched files use Prettier code style!

### PR DESCRIPTION
**Pull request by Kawaa-qwq**
Translation languages are supported in the README file and all matched files use Prettier code style!

### CODE REVIEW

**Comment by Kimi898246** [comment]
image
image

File: README_zh-TW.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+[EN](README.md) | [简中](README_zh-CN.md) | 繁中
+
+# 客戶端修補通知
+
+對於遊戲版本為 2.8 及以上時，使用 Grasscutter 啟動時，Cultivation 會自動為您的遊戲客戶端製作一個小補丁，並在關閉遊戲時恢復它。 從理論上講，你應該是完全安全的，但是不明確**如果您使用它連接到官方服務器，修改遊戲客戶端可能會導致封號**，但可能性是非常小的，並且從未接到發生過此類情況的問題，但存在這種可能性！
```
Comment (Raymonf) [inline README_zh-TW.md:5]: * 補丁 -> 修補
* 服務器 -> 伺服器

File: README_zh-TW.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+# Cultivation
+
+一個遊戲啟動器，旨在輕松將某動漫遊戲的流量代理到私人服務器。
```
Comment (Raymonf) [inline README_zh-TW.md:9]: 服務器 -> 伺服器

File: README_zh-TW.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+- `yarn build --debug`
+
+### 代碼格式化與糾錯
+
+格式化:
+
+- `yarn format`
+
+糾錯, 修復（一些）錯誤:
```
Comment (Raymonf) [inline README_zh-TW.md:58]: 標點符號不一致

File: README_zh-TW.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+- `yarn lint`, `yarn lint:fix`
+
+### 生成更新項目
+
+- 將 `TAURI_PRIVATE_KEY` 添加到環境變量，其中包含私鑰的路徑。
```
Comment (Raymonf) [inline README_zh-TW.md:64]: 環境變量 -> 環境變數

**Review by Raymonf — CHANGES_REQUESTED** [review]
looks ok but needs some vocab changes for the zh-TW readme

**Comment by Kawaa-qwq** [comment]
ok，i will fix it

**Comment by Kawaa-qwq** [comment]
I fix it

**Pull request merged**